README.md

```diff
@@ -1,6 +1,87 @@
-# csi-proxy
+# CSI Proxy
 
-CSI Proxy utility to enable CSI Plugins on Windows.
+CSI Proxy is a binary that exposes a set of gRPC APIs around storage operations
+over named pipes in Windows. A container, such as CSI node plugins, can mount
+the named pipes depending on operations it wants to exercise on the host and
+invoke the APIs.
+
+Each named pipe will support a specific version of an API (e.g. v1alpha1, v2beta1)
```

Comment:
Does each api group have its own named pipe? ie disk v1alpha1, smb v2?

Reply:
Correct. Example: https://github.com/kubernetes-csi/csi-proxy/blob/master/client/groups/filesystem/v1alpha1/client_generated.go

Reply:
From the hostpath specified [below](https://github.com/kubernetes-csi/csi-proxy/pull/16/files#diff-04c6e90faac2675aa89e2176d2eec7d8R70), it's not clear to me how to distinguish between the named pipes for different api groups.

Is it not possible to serve multiple rpc services on a single named pipe?